samples/dect/dect_phy/hello_dect/README.rst

```diff
@@ -105,6 +105,22 @@ Building and running
    If you, as a user, are not permitted to operate on the DECT band or cannot ensure that access rules are met, you must ensure there are no radio emissions from the devices running the sample.
    This can be done with the use of an RF chamber.
 
+   * To build with one of the provided overlays, run the following command:
+
+      .. parsed-literal::
+         :class: highlight
+
+         west build -p -b *build_target* -- -DCONFIG_OVERLAY="<your_overlay>"
```

Comment:
This is incorrect, should it not be `-DEXTRA_CONF_FILE`?
`-DCONFIG_OVERLAY` looks wrong, it will try to set the Kconfig `OVERLAY` to something

Reply:
Maybe it should have been `CONFIG_DTC_OVERLAY_FILE` or `CONFIG_EXTRA_DTC_OVERLAY_FILE`?

https://docs.zephyrproject.org/latest/build/dts/howtos.html#set-devicetree-overlays

https://docs.zephyrproject.org/latest/build/kconfig/setting.html#:~:text=CONF_FILE%2C%20EXTRA_CONF_FILE%2C%20DTC_OVERLAY_FILE%2C%20and%20EXTRA_DTC_OVERLAY_FILE

Reply:
You can also find the variables that needs to be used here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/configuring_app/cmake/index.html#providing_cmake_options